app/public/js/localSprites.js

```diff
@@ -151,7 +183,8 @@ function removeFinishedLocalSprites() {
 var localSprites = [];
 var twilightTilesImage = requireImage('/gfx/jetrel/twilight-tiles.png'),
 fireballBImage = requireImage('/gfx/fireball/fireballB.png'),
-fireballContrailAImage = requireImage('/gfx/fireball/fireballContrailA.png');
+fireballContrailAImage = requireImage('/gfx/fireball/fireballContrailA.png'),
+customBlocksAImage = requireImage('gfx/customBlocksA.png');
 
 function addLocalFallingSpikesSprite() {
```

Comment:
Maybe we should just delete this? It was a nice example for you to base your fireball stuff off of, but I don't have any plans of using this myself.